float32 arduYaw 
float32 arduThrottle
float32 arduPitch
float32 arduRoll
int32 arduArmGripOn
int32 arduArmGripOff
int32 arduStabilizeMode
int32 arduRemoteArmed
